### PR TITLE
Allow multiple checkins

### PR DESCRIPTION
**Pull request by haydenrou**
#434

### CODE REVIEW

**Comment by bklang** [comment]
Reminder from standup this morning, the Rails upgrades are steamrolling right along. I recommend rebasing this morning to pick up the Rails upgrades

**Comment by haydenrou** [comment]
Accidentally pushed the fix for #496 here

**Comment by benlangfeld** [comment]
You'll need to merge master into this PR to get CI to run properly again.

**Comment by compwron** [comment]
Sounds wonderful! I'm looking forward to trying it out in staging :)

**Review by phylum — APPROVED** [review]

**Review by lmerriam — APPROVED** [review]
Screenshots look good

**Review by compwron — APPROVED** [review]

File: .tool-versions [hunk 1]
```diff
@@ -0,0 +1 @@
+ruby 2.7.4
```
Comment (bklang) [inline .tool-versions]: These files should not be checked in because they conflict with the `.ruby-version` files

File: backend/.tool-versions [hunk 2]
```diff
@@ -0,0 +1 @@
+ruby 2.7.4
```
Comment (bklang) [inline backend/.tool-versions]: Ditto

File: frontend/.tool-versions [hunk 3]
```diff
@@ -0,0 +1 @@
+nodejs 12.22.6
```
Comment (bklang) [inline frontend/.tool-versions]: This file should not be checked in because it conflicts with the `.nvmrc` file

**Review by bklang — CHANGES_REQUESTED** [review]
Nice improvement! Only real objection is the `.tool-versions` files

**Review by phylum — APPROVED** [review]

**Review by bklang — APPROVED** [review]

**Pull request merged**